docs/source/release_notes.rst

```diff
@@ -104,11 +130,7 @@ Reference
 
 
 Version 1.1.0
-<<<<<<< HEAD
```

Comment:
merge conflict here

Reply:
Thanks @blakeinate , could  you check this again, some odd state because i don't see it and i fix it 1 day before its reporting, here https://github.com/ansible-collections/ibm_zos_core/commit/7185379bb2aad3fb1817560c063ff6834a979dc2

Reply:
Whoops that was a mess up on my part, didn't realize that was a removal not an addition.